man/org.freedesktop.systemd1.xml

```diff
@@ -560,6 +560,8 @@ node /org/freedesktop/systemd1 {
       readonly i DefaultOOMScoreAdjust = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly s CtrlAltDelBurstAction = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly u SoftRebootsCount = ...;
```

Comment:
hmm, it seems cleaner to me to use singular for this. i.e. SoftRebootCount

Reply:
Count or Counter?

(or Cycle?)

Reply:
It's a count of how many soft reboots happened, hence plural makes more sense to me

Reply:
(eg: we have properties such as `NRestarts`, again plural, counting restarts)